### PR TITLE
AF-2264-fix_tests_on_w2k16

### PR DESCRIPTION
**Pull request by ohonchar**
In these tests `File.separator` is not necessary, because META-INF are inside jars, so the dir structure will be using this "/"

### CODE REVIEW

File: uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/M2ServletContextListener.java [hunk 1]
```diff
@@ -66,7 +66,7 @@
     private static final String LIB_FOLDER = "lib";
     private static final String JAR_ARTIFACT = "jar";
     private static final String JARS_FOLDER = File.separator + WEB_INF_FOLDER + File.separator + LIB_FOLDER + File.separator;
-    private static final String MAVEN_META_INF = "META-INF" + File.separator + "maven";
+    private static final String MAVEN_META_INF = "META-INF/maven";
```
[the remark this answers is not in the document]
Comment (ohonchar) [reply uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/M2ServletContextListener.java]: @domhanak yes i tested on Windows and Linux, so it is pass without any errors.
Thnx

**Comment by domhanak** [comment]
@pefernan could you please review?

**Comment by domhanak** [comment]
Roses are red,
violets are blue,
Jenkins execute full downstream build!

**Comment by domhanak** [comment]
Jenkins execute full downstream build!

**Comment by domhanak** [comment]
@ohonchar My concern with this `In these tests File.separator is not necessary, because META-INF are inside jars, so the dir structure will be using this "/"`
Is that you are saying `in tests` but the class is not test but regular one.

**Comment by domhanak** [comment]
@desmax74 Could you please quickly look and see if you are okay with this change?

**Comment by desmax74** [comment]
@domhanak if the tests works both on win and linux are ok

**Comment by domhanak** [comment]
Roses are red,
violets are blue,
Jenkins execute full downstream build!

**Comment by domhanak** [comment]
Jenkins execute full downstream build!

**Comment by domhanak** [comment]
@porcelli what do you think about these changes?

**Comment by domhanak** [comment]
Jenkins execute full downstream build!

**Comment by porcelli** [comment]
as long it works on windows (as it should). i'm ok with that

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by domhanak** [comment]
@pefernan Could we please merge this?

**Pull request merged**